lib/puppet/file_system/windows.rb

```diff
@@ -100,4 +110,18 @@ def raise_if_symlinks_unsupported
     end
   end
 
+  def glob_expands_paths
```

Comment:
Is this method private to the Windows file system? if not, it should have some documentation.